msteams-platform/bots/how-to/rate-limit.md

```diff
@@ -108,10 +108,12 @@ You can also handle rate limit using the per bot per thread limit.
 
 ## Per bot per thread limit
 
->[!NOTE]
-> Message splitting at the service level results in higher than expected requests per second (RPS). If you are concerned about approaching the limits, you must implement the [backoff strategy](#backoff-example). The values provided in this section are for estimation only.
+The per bot per thread limit controls the traffic that a bot is allowed to generate in a single conversation. A conversation is 1:1 between bot and user, a group chat, or a channel in a team. So, if the application sends one bot message to each user, it does not throttle.
```

Comment:
Suggested correction:
...in a team. Therefore, if the application sends...

What is not throttled? It is not coming out clearly.